compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIELinalgFunctionOutlining.cpp

```diff
@@ -65,6 +64,19 @@ static FailureOr<func::FuncOp> outline(IRRewriter &rewriter, ModuleOp moduleOp,
   // arguments.
   Operation *clonedComputeOp = rewriter.clone(*computeOp, operandMap);
 
+  if (noAliasFinalArg) {
+    ArrayRef<BlockArgument> args = func.getArguments();
+    // Find the first MemRefType argument, starting at the end.
```

Comment:
Can't we just put `noalias` on all the arguments?

Reply:
We could put it on all memref args. ukernels have memref and index args (ukernels have offset arguments) so would at least need to filter out the non tensor indices.

Reply:
Yeah, that would better then I think as it would avoid the hardcoding on the last argument